src/api/python/speechd_config/paths.py.in

```diff
@@ -2,3 +2,6 @@ SPD_CONF_ORIG_PATH="@spdconforigdir@"
 SPD_CONF_PATH="@spdconfdir@"
```

Comment:
Maybe I should also rename this file but I am not sure if it is not a part of public API. And also, these files are typically named `config.h` or `config.py` for Python but that name is taken.

Reply:
for safety, we can create another file, and make paths.py import it.

Reply:
Any ideas for name?